wallet/src/actors/worker/methods.rs

```diff
@@ -963,6 +963,26 @@ impl Worker {
         Ok(())
     }
 
+    pub fn handle_change_node_status(
+        &self,
+        status: types::StateMachine,
+        _confirmed: bool,
+        wallet: types::SessionWallet,
+        sink: types::DynamicSink,
+    ) -> Result<()> {
+        let changed_node_status = status;
+        log::debug!(
+            "The node has chaged its status into {:?}",
+            changed_node_status
```

Comment:
I think `changed_node_status` variable is not needed?

```suggestion
            status
```